server/plugin/webhook.go

```diff
@@ -5,15 +5,13 @@ import (
 	"crypto/hmac"
 	"crypto/sha1" //nolint:gosec // GitHub webhooks are signed using sha1 https://developer.github.com/webhooks/.
 	"encoding/hex"
+	"github.com/google/go-github/v37/github"
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/microcosm-cc/bluemonday"
 	"io/ioutil"
 	"net/http"
 	"strings"
 	"time"
-
-	"github.com/microcosm-cc/bluemonday"
-
-	"github.com/google/go-github/v31/github"
-	"github.com/mattermost/mattermost-server/v5/model"
```

Comment:
This seems like an unrelated change

Reply:
Its due to gofmt.

Reply:
@sibasankarnayak your thoughts on this ? I see that it's different from master in terms of the order in which they are imported.